cmd/minishift/cmd/start_preflight.go

```diff
@@ -38,28 +39,30 @@ const (
 
 // preflightChecksAfterStartingHost is executed before the startHost function.
 func preflightChecksBeforeStartingHost() {
+	driverErrorMessage := "See the 'Setting Up the Driver Plug-in' topic for more information"
```

Comment:
IMO, documentation link should be given here. https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html

Reply:
Also, since `getting-started/setting-up-driver-plugin` doesn't have mention of being member of HyperV group either we need to mention there that one need to part of hyper-v group or point to https://docs.openshift.org/latest/minishift/troubleshooting/troubleshooting-driver-plugins.html#insufficient-privileges documentation.

Reply:
Well, none of the drivers actually specify a link online.
And maybe, the permissions should be restructured in the documentation? I do not think we should have to link to two possible locations. However, I believe the application should never have to point to a location at all.

Reply:
> And maybe, the permissions should be restructured in the documentation?

Works. We need to document this member role as part of https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html#hyper-v-driver

Reply:
Can we do this as a separate issue... Maybe use #1361 for this?

Reply:
A simple note in the driver plugin topic might do for the moment... ?

Reply:
> Maybe use #1361 for this?

Sure

Reply:
#1373

Reply:
S/"See the 'Setting Up the Driver Plug-in' topic for more information"/ Refer the documentation for ''Setting Up the Driver Plug-in' topic for more information/

Reply:
Or `Refer ''Setting Up the Driver Plug-in' topic in documentation for more information.`

Reply:
I still think the message is not descriptive enough. But I  can not come up with anything better. So we are fine at the moment.